packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2273,27 +2291,69 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
        ## Annotations for Grafana dashboard configmaps
        ##
-@@ -575,6 +942,19 @@
+@@ -574,7 +935,60 @@
+   ## Passed to grafana subchart and used by servicemonitor below
    ##
    service:
-     portName: service
+-    portName: service
++    portName: nginx-http
 +    ## Port for Grafana Service to listen on
 +    ##
 +    port: 80
 +    ## To be used with a proxy extraContainer port
 +    ##
-+    targetPort: 3000
++    targetPort: 8080
 +    ## Port to expose on each node
 +    ## Only used if service.type is 'NodePort'
 +    ##
 +    nodePort: 30950
 +    ## Service type
 +    ##
 +    type: ClusterIP
++
++  proxy:
++    image:
++      repository: rancher/library-nginx
```

Comment:
Need to do airgap testing on this PR

Reply:
left a comment on the issue for QA to track this https://github.com/rancher/rancher/issues/28495#issuecomment-696994508.